tests/UI/package.json

```diff
@@ -7,11 +7,13 @@
   },
   "scripts": {
     "linkchecker": "./node_modules/mocha/bin/mocha campaigns/tools/linkchecker.js",
-    "lint": "eslint  --fix --ignore-path .gitignore .",
+    "lint": "eslint --fix --ignore-path .gitignore .",
+    "lint-no-fix": "eslint --ignore-path .gitignore .",
```

Comment:
```suggestion
    "lint-dry-run": "eslint --ignore-path .gitignore .",
```

Reply:
On the core we use lint, and lint-fix, that would be good to collapse with others !

Reply:
Done thanks